Arrowgene.Ddon.GameServer/Handler/PawnJoinPartyMypawnHandler.cs

```diff
@@ -8,16 +8,18 @@
 using System.Collections.Generic;
 using Arrowgene.Ddon.GameServer.Party;
 using Arrowgene.Ddon.Shared.Entity;
+using Arrowgene.Ddon.GameServer.Characters;
 
 namespace Arrowgene.Ddon.GameServer.Handler
 {
     public class PawnJoinPartyMypawnHandler : GameStructurePacketHandler<C2SPawnJoinPartyMypawnReq>
     {
         private static readonly ServerLogger Logger = LogProvider.Logger<ServerLogger>(typeof(PawnJoinPartyMypawnHandler));
-
+        private readonly OrbUnlockManager _OrbUnlockManager;
```

Comment:
Why was this added? It doesn't seem to be used

Reply:
Ah, I originally thought the menu % was missing from one of these requests but I could not find the required structure. I then tried to send the NTC here but it did nothing. I will delete this line.